cmd/krel/cmd/ff.go

```diff
@@ -23,19 +23,14 @@ import (
 	"os"
 	"path/filepath"
 
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
-	"gopkg.in/src-d/go-git.v4"
-	"gopkg.in/src-d/go-git.v4/plumbing"
 
 	"k8s.io/release/pkg/command"
 	kgit "k8s.io/release/pkg/git"
 	"k8s.io/release/pkg/util"
 )
 
-var cfgFile string
-
-const defaultMasterRef string = "HEAD"
```

Comment:
We should move this into `pkg/git` instead of removing it.
There are probably several places in other tools where it might be useful to reference `HEAD`.

Reply:
Alright, I moved it over to the git package.